src/vs/code/electron-sandbox/issue/issueReporterService.ts

```diff
@@ -278,6 +278,15 @@ export class IssueReporter extends Disposable {
 		}
 	}
 
+	private async sendReporterStatus(extension: IssueReporterExtensionData): Promise<boolean[]> {
+		try {
+			const data = await this.issueMainService.$sendReporterStatus(extension.id, extension.name);
+			return data;
+		} catch (e) {
+			throw e;
```

Comment:
should this really throw? I feel like it should maybe return a sane default instead.